frontend/src/components/Navigator/index.js

```diff
@@ -92,6 +92,16 @@ function Navigator({
   const goalForEditing = watch('goalForEditing');
   const activityRecipientType = watch('activityRecipientType');
   const isGoalsObjectivesPage = page.path === 'goals-objectives';
+  const recipients = watch('activityRecipients');
+  const isRecipientReport = activityRecipientType === 'recipient';
+
+  const grantIds = isRecipientReport ? recipients.map((r) => {
```

Comment:
Why did we need to add this?

Reply:
See line 174, needed to be passed up to the API

Reply:
Was it not working before?

Reply:
I added it to resolve an error, yeah